Firestore/Source/API/FIRFirestore.mm

```diff
@@ -80,6 +80,35 @@ @implementation FIRFirestore {
   return instances;
 }
 
++ (void)load {
```

Comment:
This would likely be better off in `+initialize` rather than `+load`. If a user never calls Firestore we shouldn't register.

Reply:
Good call, will change.